src/ext/java/org/opentripplanner/ext/debug_client_geocoder/LuceneIndex.java

```diff
@@ -0,0 +1,265 @@
+package org.opentripplanner.ext.debug_client_geocoder;
```

Comment:
Even though I like using `_` in names, we have a decision on following the standard (Google, Java code-style). So, maybe just `geocoder`as package name is better. Strictly, it could also be used with other clients ?